package/kedro_viz/server.py

```diff
@@ -580,10 +580,20 @@ def _get_task_metadata(node):
 def _get_dataset_metadata(node):
     dataset = node["obj"]
     if dataset:
-        dataset_metadata = {
-            "type": f"{dataset.__class__.__module__}.{dataset.__class__.__qualname__}",
-            "filepath": str(dataset._describe().get("filepath")),
-        }
+        filepath = str(dataset._describe().get("filepath"))
+        if dataset.__class__.__qualname__ == "PlotlyDataSet":
+            with open(filepath) as file:
+                plotly_json = json.load(file)
+            dataset_metadata = {
+                "type": f"{dataset.__class__.__module__}.{dataset.__class__.__qualname__}",
+                "filepath": filepath,
+                "plot": plotly_json,
+            }
+        else:
+            dataset_metadata = {
+                "type": f"{dataset.__class__.__module__}.{dataset.__class__.__qualname__}",
+                "filepath": filepath,
+            }
```

Comment:
Could we simplify it to something like this?
```suggestion
        filepath = str(dataset._describe().get("filepath"))
        dataset_metadata = {
            "type": f"{dataset.__class__.__module__}.{dataset.__class__.__qualname__}",
            "filepath": filepath,
        }
        if dataset.__class__.__qualname__ == "PlotlyDataSet":
            with open(filepath) as file:
                plotly_json = json.load(file)
            dataset_metadata = {**dataset_metadata, "plot": plotly_json}
            
```